src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -133,7 +133,7 @@ export const useSendThorTx = ({
     ...reactQueries.thornode.inboundAddresses(),
     staleTime: 60_000,
     select: data => selectInboundAddressData(data, assetId),
-    enabled: Boolean(assetId && assetId !== thorchainAssetId),
+    enabled: Boolean((assetId && assetId !== thorchainAssetId) || (memo ?? '').includes('tcy')),
```

Comment:
We probably don't need this, got rugged by the inbound_addresses endpoint failing since THOR was down, which in turn made everything fail - retest me once THOR is fully back, we can probably remove this

Note for when tcy is live: https://stagenet-thornode.ninerealms.com/thorchain/balance/module/tcy_claim

Also: not sure where the Tx should be sent to just yet, think that's a TBD after go-live of tcy